### PR TITLE
BTRX - international Cohorts fix

### PR DESCRIPTION
**Pull request by LeoForconesi**
## Changes
- Before we build our object to be sent and save after approval, we needed a way to refresh our variable `intCohortsAnswers`, which if the user did never edited these values,  will remain empty. 
For this we created `parseIntCohorts( )` that will take all answers scattered on extra properties coming from 'Issue_review' database table, and resfresh our oject.
---

- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Get a thumb from @rushtong
- [ ] **Submitter**: Merge to develop using github's "Squash and Merge" feature
- [ ] **Submitter**: Delete branch after merge

### CODE REVIEW

File: grails-app/assets/javascripts/consentGroupReview.js [hunk 1]
```diff
@@ -91491,6 +91490,22 @@ var ConsentGroupReview = function (_Component) {
       });
       this.init();
     }
+  }, {
+    key: 'parseIntCohorts',
+    value: function parseIntCohorts() {
+      var _this3 = this;
```
Comment (rushtong) [inline grails-app/assets/javascripts/consentGroupReview.js]: the `var`s should be `let`s

File: grails-app/assets/javascripts/consentGroupReview.js [hunk 2]
```diff
@@ -91513,21 +91528,21 @@ var ConsentGroupReview = function (_Component) {
   }, {
     key: 'isCurrentUserAdmin',
     value: function isCurrentUserAdmin() {
-      var _this3 = this;
+      var _this4 = this;
```
Comment (rushtong) [inline grails-app/assets/javascripts/consentGroupReview.js]: I'm curious about why we need to rename these. Are we using these in local storage?

**Comment by LeoForconesi** [comment]
@rushtong I didn't change any of those files intentionally, it seems that the reason of those differences is the re-compiled files produced by  `webpackDev`. I'm not sure if I can fix them, will try though.

**Comment by rushtong** [comment]
@LeoForconesi - Ah ... I missed that detail. If you run the webpackProd command, my comments should be ignored. Looks like webpack made the `var`s too.

**Pull request merged**